apostrophecms/apostrophe
# Pro 3522 inline array

Currently inline array fields display labels again for each row. Change this to a more tabular view, per Stu's design:

Note that the table headers only appear once.

This is a good general improvement, so no need for an option flag.

Acceptance criteria

Headings appear on inline array fields only once at the top.

Inline array fields line up appropriately on each row as more are added.

## How to test?

You need https://github.com/apostrophecms/advanced-permission/pull/41

1. Check a piece document permissions and check the display
2. Play with the field options on advanced permissions side
   - `draggable`: `true` | `false`
   - `styles`: `'table'` | `''`
   - `inline`: `true` | `false`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
